app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -131,6 +131,7 @@ class MenuDrawerFragment : Fragment() {
                             appBuildNumber = BuildConfig.VERSION_NAME,
                             bucketIdentifier = BuildConfig.BUGTRACKER_MAIL_BUCKET_ID,
                             projectName = BuildConfig.BUGTRACKER_MAIL_PROJECT_NAME,
+                            repoGithub = BuildConfig.GITHUB_REPO,
```

Comment:
Waiting for https://github.com/Infomaniak/android-core/pull/32#discussion_r1100137393 to be resolved.